content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -994,17 +994,17 @@ Not supported
 <Admonition type="warning">
 
 **Not supported by MongoDB** <br />
-The [MongoDB connector](/concepts/database-connectors/mongodb) does not currently support the `Unsupported` type.
+The [MongoDB connector](/concepts/database-connectors/mongodb) does not support the `Unsupported` type.
 
 </Admonition>
 
-The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by the Prisma Client. `Unsupported` fields can be introspected with `prisma db pull` or created with Prisma Migrate or `db push`.
+The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by Prisma Client. Fields of type `Unsupported` can be created during Introspection with `prisma db pull` or written by hand, and created in the database with Prisma Migrate or `db push`.
```

Comment:
I may not be understanding how this works at all, so please ignore any part of this suggestion that doesn't make sense...

```suggestion
The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by Prisma Client. You can create fields of type `Unsupported` when you introspect with `db pull`, or add them manually to your Prisma schema and apply them to the database with Prisma Migrate or `db push`.
```

Reply:
This is fundamentally about 2 directions:
1. Getting them into the schema, which can be done via Introspection or manually
2. Migrating them to a database when you just have the schema, which is Migrate or `db push`.

Maybe 2 separate sentences would be better here.